src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py

```diff
@@ -0,0 +1,7 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+RESOURCE_TYPE = 'Microsoft.ContainerRegistry/registries'
+DOMAIN_SUFFIX = '.azurecr.io'
```

Comment:
This DOMAIN_SUFFIX was only used in catalog command when the user wants to list repositories in a registry that is NOT under the user's subscription. When we support different domain name suffix or even custom domains, we may want the user to enter the full URL. The CLI doesn't need this, though, since the login server can be obtained from ARM/RP.